code/datums/controllers/action_controls.dm

```diff
@@ -104,7 +104,7 @@ var/datum/action_controller/actions
 
 	proc/onRestart()			   //Called when the action restarts (for example: automenders)
 		sleep(1)
-		started = world.time
+		started = TIME
```

Comment:
Won't this need changes in other places too?

Reply:
it did, done!